common/src/main/java/com/wynntils/models/characterstate/event/CharacterDeathEvent.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.characterstate.event;
+
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.eventbus.api.Event;
+
+public class CharacterDeathEvent extends Event {
+
+    private final Vec3 deathLocation;
+
+    public CharacterDeathEvent(Vec3 deathLocation) {
+        this.deathLocation = deathLocation;
+    }
+
+    public Vec3 getDeathLocation() {
```

Comment:
Also, I think you can shorten the `deathLocation` to just `location` everywhere. It is the `CharacterDeathEvent` after all, so location cannot reasonably mean anything else than the death location...